dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/ChatHistoryExtensions.cs

```diff
@@ -21,7 +21,7 @@ public static class ChatHistoryExtensions
     /// <param name="streamingMessageContents"><see cref="IAsyncEnumerator{T}"/> list of streaming message contents</param>
     /// <returns>Returns the original streaming results with some message processing</returns>
     [Experimental("SKEXP0010")]
-    public static async IAsyncEnumerable<StreamingChatMessageContent> AddStreamingMessageAsync(this ChatHistory chatHistory, IAsyncEnumerable<OpenAIStreamingChatMessageContent> streamingMessageContents)
+    public static async IAsyncEnumerable<OpenAIStreamingChatMessageContent> AddStreamingMessageAsync(this ChatHistory chatHistory, IAsyncEnumerable<OpenAIStreamingChatMessageContent> streamingMessageContents)
```

Comment:
Why do we need this in our public API surface?

The only use I see in the current codebase is in the associated unit test.

Reply:
I don't have the context.  Nor have I changed visibility.  The only change here is noticing that the down-cast is degrading the caller context (anti-pattern) as the caller already must be passing in a set of `OpenAIStreamingChatMessageContent` objects.

By inspection, it appears that raison d'etra for this is to append the result of streaming messages to an existing chat-history object.  A common operation that doesn't need to be federated to every single caller.

Reply:
Let's leave the API as is rather than introduce the noise of a breaking change. I've created this task to see can we remove this: https://github.com/microsoft/semantic-kernel/issues/5740

Reply:
I agree, if the change is not related to the purpose of this PR and it's forcing us to create suppression file, it's probably better to avoid it here and track it as part of separate task.